core/__tests__/actions/navigation.ts

```diff
@@ -79,7 +80,11 @@ describe("actions/navigation", () => {
         password: "P@ssw0rd!",
         email: "peach@example.com",
       });
+
       process.env.GROUPAROO_RUN_MODE = "cli:config";
+      process.env.GROUPAROO_CONFIG_DIR = `${os.tmpdir()}/test/${
+        process.env.JEST_WORKER_ID
+      }/config/navigation`;
```

Comment:
This will make a directory in `/tmp` to store the config files... outside of the repo :D